cmd/access/access_node_builder.go

```diff
@@ -59,26 +61,28 @@ type AccessNodeBuilder interface {
 // For a node running as a standalone process, the config fields will be populated from the command line params,
 // while for a node running as a library, the config fields are expected to be initialized by the caller.
 type AccessNodeConfig struct {
-	staked                       bool
-	stakedAccessNodeIDHex        string
-	unstakedNetworkBindAddr      string
-	blockLimit                   uint
-	collectionLimit              uint
-	receiptLimit                 uint
-	collectionGRPCPort           uint
-	executionGRPCPort            uint
-	pingEnabled                  bool
-	nodeInfoFile                 string
-	apiRatelimits                map[string]int
-	apiBurstlimits               map[string]int
-	rpcConf                      rpc.Config
-	ExecutionNodeAddress         string // deprecated
-	HistoricalAccessRPCs         []access.AccessAPIClient
-	logTxTimeToFinalized         bool
-	logTxTimeToExecuted          bool
-	logTxTimeToFinalizedExecuted bool
-	retryEnabled                 bool
-	rpcMetricsEnabled            bool
+	staked                              bool
+	stakedAccessNodeIDHex               string
+	stakedAccessNodeAddress             string
+	stakedAccessNodeNetworkingPublicKey string
```

Comment:
Commenting here to highlight changes

Reply:
thanks. new fields - `stakedAccessNodeAddress` and `stakedAccessNodeNetworkingPublicKey`